tests/installation/addon_products_sle.pm

```diff
@@ -68,30 +68,28 @@ sub run() {
             send_key 'alt-u';                                                   # select URL field
             type_string get_var("ADDONURL_$uc_addon");                          # repo URL
             send_key $cmd{next};
-            if (get_var("ADDONS")) {
-                if (get_var("BETA_$uc_addon")) {
-                    assert_screen "addon-betawarning-$addon";
+            my @tags = ("addon-products", "addon-betawarning-$addon", "addon-license-$addon");
+            check_screen(\@tags, 90);
```

Comment:
The "90s" was taken from existing code in the same module "assert_screen 'addon-products', 90;".
The possible reason for 90s timeout may be longer time is required to load addon repo from specified URL in case network is slow.